### ISSUE TITLE
Flash message is burried on the admin page

### ISSUE THREAD

**Issue by jcoyne** [projecthydra-labs/hyrax#595]
<img width="1326" alt="screen shot 2017-03-20 at 12 20 55 pm" src="https://cloud.githubusercontent.com/assets/92044/24112522/cfe5ff42-0d67-11e7-803b-f2b45d4c0d1a.png">

**Comment by jcoyne** [comment]
cc @ggeisler

**Comment by ggeisler** [comment]
@jcoyne Can we give the flash message div the same Bootstrap column classes as the `.main-content` div:

``` col-xs-12 col-sm-9 col-sm-push-3 col-lg-10 col-lg-push-2```

but **only for the admin UI**? That is, without affecting flash messages shown (I assume there can be, though not sure) on the non-admin side? If so, I think that would handle it fine.

If not, and there is no other clear solution, we might want to reconsider the fixed position approach I used here. The important part is the clean-looking sidebar, not the fixed position aspect.  I only did that as a way to get a clean-looking sidebar without having to modify the current HTML structure in the Hyrax layout view.

For example, in Spotlight the main content area and the sidebar are defined cols within the same row, so even if the main content area is shorter than the sidebar, the sections remain distinct:

![curation_-_tags___douglas_menuez_-_online_exhibits](https://cloud.githubusercontent.com/assets/101482/24115089/784ddd52-0d5f-11e7-9e41-e4d82ffaeb71.png)

In contrast, in Hyrax the main content and sidebar sections are put inside a `.fluid-container` div and positioned using `col-push` and `col-pull` classes, not with a `.row`.
